lib/yggdrasil/player.ex

```diff
@@ -12,9 +13,8 @@ defmodule Yggdrasil.Player do
     Supervisor.add_player user_id, [owner_pid, push_msg]
   end
 
-  def run_cmd(user_id, cmd = %Message{type: :command}) do
-    player_pid = Registry.get_player user_id
-    GenServer.cast(player_pid, {:run_cmd, cmd})
+  def run_cmd(user_id, cmd) do
+    Command.execute cmd, user_id
```

Comment:
Well something will need to do it but I think I see your point. My original intention was that the `user_id` would be able to help us identify the room they are in, their character, anything else the command might need to perform it's logic.

I'm certainly not married to the design. Perhaps another idea is that the room executes all commands and passes in the room struct and character id then it returns the updated room struct? Then when you issue a command the player process would lookup the room and forward the command with who submit it. Only sketchy part is we wouldn't want to crash the room if something was wrong with the command. I suppose the room could spawn a process for the command. Yeah I don't know. More thought will need to be done.